### PR TITLE
kubadm/cmd: wording and punctuation fixes

### PR DESCRIPTION
**Pull request by neolit123**
**What this PR does / why we need it**:
This is a follow-up PR from me fixing more wording & punctuation in the `kubeadm/app/cmd` namespace. I think it makes the output shown to the user clearer and better formatted.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
NONE

**Special notes for your reviewer**:
Please let me know if you want anything amended. Certain things might be a subject to an argument, so if the maintainers want them kept the way they are ATM, i would comply.

**Release note**:

```release-note
NONE
```

Lubomir (VMware)

### CODE REVIEW

**Comment by neolit123** [comment]
/test pull-kubernetes-unit

the CI seems broken lately.

File: cmd/kubeadm/app/cmd/token.go [hunk 1]
```diff
@@ -53,25 +53,24 @@ func NewCmdToken(out io.Writer, errW io.Writer) *cobra.Command {
 	var dryRun bool
 	tokenCmd := &cobra.Command{
 		Use:   "token",
-		Short: "Manage bootstrap tokens.",
+		Short: "Manage Bootstrap Tokens.",
 		Long: dedent.Dedent(`
-			This command will manage Bootstrap Token for you.
-			  Please note this usage of this command is optional, and mostly for advanced users.
+			This command will manage a Bootstrap Token for you.
```
Comment (mattmoyer) [inline cmd/kubeadm/app/cmd/token.go]: Maybe just "This command manages bootstrap tokens. It is optional and needed only for advanced use cases."?

File: cmd/kubeadm/app/cmd/token.go [hunk 2]
```diff
@@ -53,25 +53,24 @@ func NewCmdToken(out io.Writer, errW io.Writer) *cobra.Command {
...
+			Please note that the usage of this command is optional and for advanced users.
 
 			In short, Bootstrap Tokens are used for establishing bidirectional trust between a client and a server.
-			A Bootstrap Token can be used when a client (for example a node that's about to join the cluster) needs
+			A Bootstrap Token can be used when a client (for example a node that is about to join the cluster) needs
```
Comment (mattmoyer) [inline cmd/kubeadm/app/cmd/token.go]: Nit: Maybe we can drop the weird title casing on "Bootstrap Token" in this section and just say "bootstrap token"? I don't think it necessarily needs to be capitalized and it scans weird for me.

File: cmd/kubeadm/app/cmd/token.go [hunk 3]
```diff
@@ -94,10 +93,10 @@ func NewCmdToken(out io.Writer, errW io.Writer) *cobra.Command {
 	var description string
 	createCmd := &cobra.Command{
 		Use:   "create [token]",
-		Short: "Create bootstrap tokens on the server.",
+		Short: "Create Bootstrap Tokens on the server.",
```
Comment (mattmoyer) [inline cmd/kubeadm/app/cmd/token.go]: Nit: I like being consistent, but I'd rather go with lowercase.

File: cmd/kubeadm/app/cmd/token.go [hunk 4]
```diff
@@ -116,9 +115,9 @@ func NewCmdToken(out io.Writer, errW io.Writer) *cobra.Command {
 		},
 	}
 	createCmd.Flags().DurationVar(&tokenDuration,
-		"ttl", kubeadmconstants.DefaultTokenDuration, "The duration before the token is automatically deleted (e.g. 1s, 2m, 3h). 0 means 'never expires'.")
+		"ttl", kubeadmconstants.DefaultTokenDuration, "The duration before the token is automatically deleted (e.g. 1s, 2m, 3h). If set to '0', the token will never expire.")
 	createCmd.Flags().StringSliceVar(&usages,
-		"usages", kubeadmconstants.DefaultTokenUsages, "The ways in which this token can be used. Valid options: [signing,authentication].")
+		"usages", kubeadmconstants.DefaultTokenUsages, "The ways in which this token can be used. Valid options are 'signing' and 'authentication'.")
```
Comment (mattmoyer) [inline cmd/kubeadm/app/cmd/token.go]: We might want to include some note about how multiple usages can be specified (with commas or by passing `---usages` multiple times, IIRC).

**Review by mattmoyer — COMMENTED** [review]
A few nits, but this is great!

File: cmd/kubeadm/app/cmd/token.go (hunk 2 above)
Comment (neolit123) [reply cmd/kubeadm/app/cmd/token.go]: @mattmoyer 
yeah, the capitalization of "Boostrap Token(s)" was consistent in a number of places. should i just make it lower case (i.e. "bootsrap token(s)") everywhere?

File: cmd/kubeadm/app/cmd/token.go (hunk 3 above)
Comment (neolit123) [reply cmd/kubeadm/app/cmd/token.go]: ok, making this lowercase everywhere.

**Comment by neolit123** [comment]
thanks for the review @mattmoyer
i will amend this later today.

**Comment by neolit123** [comment]
@mattmoyer amend complete. i've also:
- reflected some of the changes in `phases` and `upgrade` for consistency and added punctuation where needed.
- made `Bootstrap Tokes` lowercase everywhere

File: cmd/kubeadm/app/cmd/token.go (hunk 4 above)
Comment (neolit123) [reply cmd/kubeadm/app/cmd/token.go]: this became:
```
Describes the ways in which this token can be used. You can pass --usages
multiple times or provide a comma separated list of options. Valid options
are 'signing' and 'authentication'.
```

**Comment by neolit123** [comment]
hmm, i wonder if `-unit` is failing because of my changes:
```
I1027 21:54:06.285] FAIL	k8s.io/kubernetes/cmd/kubeadm/app/cmd/upgrade	1.225s
```
doesn't seem like so.

**Comment by neolit123** [comment]
bump, is the CI broken here?

**Comment by mattmoyer** [comment]
I was able to reproduce the `pull-kubernetes-unit` failure locally and it gave more helpful (still somewhat inscrutable) error: https://gist.github.com/mattmoyer/9eda332d4b2d85cc0f9b555c1558c011

I think you just need to update the strings in that test to match your changes: https://gist.github.com/mattmoyer/a34d1e0717653139ae1819fe62a6e4d0

**Comment by neolit123** [comment]
thanks, i will try.

**Comment by neolit123** [comment]
@mattmoyer 

i've applied the patch to `plan_test.go` and locally all goes well:

```
lubo-it@luboitvbox:~/go/src/k8s.io/kubernetes$ make test

<snip>

ok  	k8s.io/kubernetes/pkg/util/metrics	0.003s  <----------------

<snip>

ok  	k8s.io/kubernetes/vendor/k8s.io/sample-apiserver/pkg/admission/wardleinitializer	0.047s
ok  	k8s.io/kubernetes/vendor/k8s.io/sample-apiserver/pkg/apis/wardle/install0.058s
ok  	k8s.io/kubernetes/vendor/k8s.io/sample-apiserver/pkg/apiserver	0.077s
lubo-it@luboitvbox:~/go/src/k8s.io/kubernetes$ 

lubo-it@luboitvbox:~/go/src/k8s.io/kubernetes$ go test -v k8s.io/kubernetes/cmd/kubeadm/app/cmd/upgrade -run TestPrintAvailableUpgrades$
=== RUN   TestPrintAvailableUpgrades
--- PASS: TestPrintAvailableUpgrades (0.00s)
PASS
ok  	k8s.io/kubernetes/cmd/kubeadm/app/cmd/upgrade	0.072s

````

but the online CI test is saying:
```
I1108 23:10:48.365] FAIL	k8s.io/kubernetes/pkg/util/metrics	0.024s
```

so `metrics` isn't failing locally for me.

**Comment by neolit123** [comment]
```
lubo-it@luboitvbox:~/go/src/k8s.io/kubernetes/pkg/util/metrics$ go test -v k8s.io/kubernetes/pkg/util/metrics -run TestRegisterMetricAndTrackRateLimiterUsage$
=== RUN   TestRegisterMetricAndTrackRateLimiterUsage
--- PASS: TestRegisterMetricAndTrackRateLimiterUsage (0.00s)
PASS
ok  	k8s.io/kubernetes/pkg/util/metrics	0.005s
```

**Comment by neolit123** [comment]
@mattmoyer tests passed.

**Comment by mattmoyer** [comment]
/lgtm

Thanks for slogging through all this @neolit123!

**Comment by neolit123** [comment]
thank you.

**Comment by neolit123** [comment]
> @neolit123 PR needs rebase

great timing... :)

**Comment by neolit123** [comment]
changes rebased.

File: cmd/kubeadm/app/cmd/upgrade/upgrade.go [hunk 5]
```diff
@@ -50,12 +50,12 @@ func NewCmdUpgrade(out io.Writer) *cobra.Command {
...
 	cmd.PersistentFlags().BoolVar(&flags.allowExperimentalUpgrades, "allow-experimental-upgrades", flags.allowExperimentalUpgrades, "Show unstable versions of Kubernetes as an upgrade alternative and allow upgrading to an alpha/beta/release candidate versions of Kubernetes.")
 	cmd.PersistentFlags().BoolVar(&flags.allowRCUpgrades, "allow-release-candidate-upgrades", flags.allowRCUpgrades, "Show release candidate versions of Kubernetes as an upgrade alternative and allow upgrading to a release candidate versions of Kubernetes.")
-	cmd.PersistentFlags().BoolVar(&flags.printConfig, "print-config", flags.printConfig, "Whether the configuration file that will be used in the upgrade should be printed or not.")
-	cmd.PersistentFlags().BoolVar(&flags.skipPreFlight, "skip-preflight-checks", flags.skipPreFlight, "Skip preflight checks normally run before modifying the system")
+	cmd.PersistentFlags().BoolVar(&flags.printConfig, "print-config", flags.printConfig, "Speficies whether the configuration file that will be used in the upgrade should be printed or not.")
```
Comment (mattmoyer) [inline cmd/kubeadm/app/cmd/upgrade/upgrade.go]: Whoops, sorry I didn't catch this before but `s/Speficies/Specifies/` here.

**Review by mattmoyer — CHANGES_REQUESTED** [review]

File: cmd/kubeadm/app/cmd/upgrade/upgrade.go (hunk 5 above)
Comment (neolit123) [reply cmd/kubeadm/app/cmd/upgrade/upgrade.go]: my bad, updated.

**Review by luxas — APPROVED** [review]
Great contribution!
/lgtm
/approve

Many thanks for the reviewing work done here @mattmoyer :clap:!

**Comment by neolit123** [comment]
fixed this too:
```
The Kubelet was informed of new the secure connection details.
```
`s/new the/the new/`

**Pull request merged**